Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.2
+MAINTAINER dengnan@google.com vmarmol@google.com vishnuk@google.com jimmidyson@gmail.com
+
+WORKDIR /gopath/src/github.com/google/cadvisor
+COPY    . /gopath/src/github.com/google/cadvisor
+
+RUN apk add --update --repository=http://dl-4.alpinelinux.org/alpine/edge/community/ -t build-deps go sqlite-dev btrfs-progs-dev bash linux-headers make git gcc g++ && \
```

Comment:
So this change makes the prod image larger as it still includes source in the final image - no way to delete that unfortunately as it's in a lower layer. If we want to keep small image size without source then perhaps we should go with an external build via jenkins/circleci? Pretty easy to set up.